django_spiff_workflow/templates/user_task_form.html

```diff
@@ -1,22 +1,35 @@
 {% load crispy_forms_tags %}
 {% load static %}
-<form method="post"
-      action="{% url 'workflow:run' workflow.id task.id %}"
-      class="needs-validation">
-    {% csrf_token %}
-    <div class="card">
-        <div class="card-body">
-            {% for field in form %}
-                {% if forloop.counter0|divisibleby:3 %}<div class="form-group row">{% endif %}
-                    <div class="col-lg-4" bis_skin_checked="1">
-                        {{ field|as_crispy_field }}
-                        {% if field.errors %}<div>{{ field.errors }}</div>{% endif %}
-                    </div>
-                    {% if forloop.counter0|add:1|divisibleby:3 %}</div>{% endif %}
-            {% endfor %}
-            <div class="form-group row">
-                <button type="submit" class="btn btn-danger">Submit</button>
-            </div>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet"
+          type="text/css"
+          href="{% static 'css/user_task_form.css' %}" />
+  
+</head>
+<body>
```

Comment:
This is the complete HTML page but the template will be part of the another html page. This should only have the form